README.md

```diff
@@ -11,27 +11,89 @@ Cisco Catalyst WAN SDK is a package for creating simple and parallel automatic r
 pip install catalystwan
 ```
 
-## Session usage example
-Our session is an extension to `requests.Session` designed to make it easier to communicate via API calls with SDWAN Manager. We provide ready to use authentication, you have to simply provide the Manager url, username and password as as if you were doing it through a GUI. 
+## Manager Session
+In order to execute SDK APIs **ManagerSession** needs to be created. The fastest way to get started is to use `create_manager_session()` method which configures session, performs authentication for given credentials and returns **ManagerSession** instance in operational state. **ManagerSession** provides a collection of supported APIs in `api` instance variable.
```

Comment:
Consider emphasizing the benefits of utilizing the endpoints variable, especially in scenarios where direct API implementations are not feasible or preferred.

Reply:
It might become to cluttered for quickstart - api container is suggested way of using SDK. I will Add Endpoint container information below api usage examples.

Reply:
Sounds good 👍